File: donation-system-web/src/types/ageRange.ts
export interface AgeRangeResponse {
  id: string;
  name: string;
  minAge: number;
  maxAge: number;
  active: boolean;
  createdAt: string;
  updatedAt?: string;
}

export interface CreateAgeRangeRequest {
  name: string;
  minAge: number;
  maxAge: number;
  active: boolean;
}

export interface UpdateAgeRangeRequest {
  name: string;
  minAge: number;
  maxAge: number;
  active: boolean;
}

File: APISolidarityManager/APISolidarityManager/Entities/AgeRange.cs
namespace APISolidarityManager.Models
{
    public class AgeRange
    {
        public Guid Id { get; set; }
        public string Name { get; set; } = null!;
        public int MinAge { get; set; }
        public int MaxAge { get; set; }
        public bool Active { get; set; } = true;
        public DateTime CreatedAt { get; set; }
        public DateTime? UpdatedAt { get; set; }

        public ICollection<NeedRule> NeedRules { get; set; } = new List<NeedRule>();
    }
}
